### PR TITLE
fix label+input/select styling

### PR DESCRIPTION
**Pull request by ulferts**
Fixes label/input field associations in various places. This also include the action menus.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://feature-foundation-apps-framework-inline-label.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://feature-foundation-apps-framework-inline-label.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/views/repositories/_navigation.html.erb [hunk 1]
```diff
@@ -33,21 +33,33 @@ See doc/COPYRIGHT.rdoc for more details.
 
 <% content_for :action_menu_specific do %>
   <!--[if gt IE 8]><!-->
```
Comment (myabc) [inline app/views/repositories/_navigation.html.erb]: @ulferts not directly related to this task, do we still need this conditional comment?

Comment (ulferts) [inline app/views/repositories/_navigation.html.erb]: no, will remove

File: app/assets/stylesheets/content/_legacy_actions.sass [hunk 2]
```diff
@@ -94,3 +94,14 @@ ul.legacy-actions-more
 
 .message-reply-menu
   @include contextual(-39px)
+
+.action-menu-inline-label
```
Comment (myabc) [inline app/assets/stylesheets/content/_legacy_actions.sass]: @ulferts confusingly we're using `.action-menu` for the new (Angular-based) context menu. Some time ago we renamed the classes associated with the older menu to `legacy-actions-*`

also, I haven't documented it yet, but I'm basing my CSS naming on [the ideas expressed in this article](http://www.hagenburger.net/BLOG/Modular-CSS-Class-Names.html) (which in turn are based roughly on the [BEM](https://bem.info/) methodology). In this vein, the class naming would probably end up looking like: `legacy-actions--inline-label`.

**Comment by myabc** [comment]
:+1:

**Pull request merged**